core/domain/prod_validation_jobs_one_off.py

```diff
@@ -529,6 +544,117 @@ def validate(cls, item):
         cls._validate_post_commit_is_private(item)
 
 
+class BaseUserModelValidator(BaseModelValidator):
+    """Class for validating BaseUserModels."""
+
+    @classmethod
+    def _get_model_id_regex(cls, unused_item):
+        return '^\\d*$'
```

Comment:
Can you explain the justification for this regex? Why * and not +?

Reply:
My bad, it will be `+`. Fixed this.